src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/sendtofirsttier/SendToFirstTierAboutToSubmitHandlerTest.java

```diff
@@ -84,6 +85,7 @@ void givenAInvalidCallbackType_thenReturnFalse() {
     @ParameterizedTest
     @EnumSource(SendToFirstTierActions.class)
     void shouldReturnWithoutError(SendToFirstTierActions action) {
+        caseData.setHearings(List.of(Hearing.builder().value(HearingDetails.builder().build()).build()));
```

Comment:
Ideally we don't want to be using nested builders, cleaner to separate them out even if no params being set/passed.

Reply:
done